### DISCUSSION TITLE
Video Tutorial?

### DISCUSSION THREAD

**Discussion by JamesM92** [docker-mailserver/docker-mailserver#2725] [General]
Does anybody know of a video tutorial on youtube walking through the whole setup process?

I've tried searching "Docker-mailserver" and i don't find anything, but i come across dozens of videos for mailcow, mailu, redmail, zimbra, mailous, and others, but nobodies made a tutorial for this.

**Comment by polarathene** [comment]
Despite having more stars than the individual projects on Github, we're only a few maintainers volunteering time on keeping the project in a good working state.

Our docs could do with more love, and perhaps communities find the other projects easier to setup or they are large enough to have users create video content.

There are some [text-based tutorials](https://docker-mailserver.github.io/docker-mailserver/edge/examples/tutorials/basic-installation/) in our docs (also in the [blogposts section](https://docker-mailserver.github.io/docker-mailserver/edge/examples/tutorials/blog-posts/)), they may be not exactly matching what you want, but hopefully not out of date.

If you do find a video tutorial, feel free to contribute it to our docs for the benefit of other users to find :)

---

While not directly related, frequency of contributions (and number of contributors) may somewhat play into it.

I know at least one of the other ones are financially backed, but I haven't looked into contribution activity or how many contributors are regularly involved in those projects. We are based on shell scripts vs programming languages like Python that is common with the others. So it likely affects friction to contribute.

We're also shipping everything in one container while at least some of the projects (_that are docker focused at least_) are more modular and may have broader documentation?
